cmd/operator/main.go

```diff
@@ -90,9 +97,7 @@ func main() {
 	op, err := operator.New(logger, cfg, metrics, operator.Options{
 		ProjectID:               *projectID,
 		Cluster:                 *cluster,
-		Location:                *location,
```

Comment:
Why remove location initializing here?

Reply:
Something happened in this file and I'm not sure what (: